examples/logging-log4j-appender/config/logging.properties

```diff
@@ -0,0 +1,70 @@
+# Additional loggers to configure (the root logger is always configured)
+loggers=sun.rmi,io.jaegertracing.Configuration,org.jboss.as.config,com.arjuna
+
+logger.level=INFO
+logger.handlers=FILE,CONSOLE,log4j-file
+
+logger.sun.rmi.level=WARN
+logger.sun.rmi.useParentHandlers=true
+
+logger.io.jaegertracing.Configuration.level=WARN
+logger.io.jaegertracing.Configuration.useParentHandlers=true
+
+logger.org.jboss.as.config.level=DEBUG
+logger.org.jboss.as.config.useParentHandlers=true
+
+logger.com.arjuna.level=WARN
+logger.com.arjuna.useParentHandlers=true
+
+handler.CONSOLE=org.jboss.logmanager.handlers.ConsoleHandler
+handler.CONSOLE.level=INFO
+handler.CONSOLE.formatter=COLOR-PATTERN
+handler.CONSOLE.properties=autoFlush,target
+handler.CONSOLE.autoFlush=true
+handler.CONSOLE.target=SYSTEM_OUT
+
+handler.FILE=org.jboss.logmanager.handlers.PeriodicRotatingFileHandler
+handler.FILE.level=ALL
+handler.FILE.formatter=PATTERN
+handler.FILE.properties=autoFlush,append,fileName,suffix
+handler.FILE.constructorProperties=fileName,append
+handler.FILE.autoFlush=true
+handler.FILE.append=true
+handler.FILE.fileName=${jboss.server.log.dir}/server.log
+handler.FILE.suffix=.yyyy-MM-dd
+
+handler.log4j-file=org.jboss.as.logging.logmanager.Log4jAppenderHandler
+handler.log4j-file.module=org.jboss.as.logging
+handler.log4j-file.level=ALL
+handler.log4j-file.formatter=json
+handler.log4j-file.postConfiguration=activate
+handler.log4j-file.properties=appender,enabled,dummy
+handler.log4j-file.appender=log4j-file
+handler.log4j-file.enabled=true
+handler.log4j-file.dummy=
```

Comment:
Is this one required?

Reply:
It does seem to be required. I'm trying to remember why though :) I'll look into and either remove it or document as to why it's required.

Reply:
Okay I updated the documentation explaining it.